tools/travis_script.py

```diff
@@ -34,12 +35,15 @@
     '--target-arch=i686'
 ]
 
+
 def run_docker():
     ex.check_run_cmd('docker', ['run', '-dit', '--privileged',
-                     '--name', DOCKER_NAME, '-v',
-                     '%s:%s' % (TRAVIS_BUILD_PATH, DOCKER_SHADOW_NODE_PATH),
-                     '--add-host', 'test.mosquitto.org:127.0.0.1',
-                     'iotjs/ubuntu:0.8'])
+                                '--name', DOCKER_NAME, '-v',
+                                '%s:%s' % (TRAVIS_BUILD_PATH,
+                                           DOCKER_SHADOW_NODE_PATH),
+                                '--add-host', 'test.mosquitto.org:127.0.0.1',
```

Comment:
We should add example.com here?

Reply:
There is no mock http/https server setup in container yet, so no host could be added.